src/homepage/howTo/custom-listeners.html

```diff
@@ -0,0 +1,237 @@
+---
+layout: howTo
+---
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements. See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership. The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+<!-- Main -->
+<div id="main">
+
+    <!-- Introduction -->
+    <section id="intro" class="main special">
+        <div class="">
+            <div class="content align-left">
+                <header class="major">
+                    <h1><b>Configure Custom Listeners</b></h1>
+                </header>
+
+                <p>
+                    This document will help you understand what is Mailbox Listener, then have a chance to implement your own Mailbox Listener
+                    and configure to use it in James server.
+                </p>
+
+                <ul>
+                    <li>What is Mailbox Listener</li>
+                    <li>How to use a custom Mailbox Listener in James</li>
+                </ul>
+
+                <header class="major">
+                    <h2><b>What is Mailbox Listener</b></h2>
+                </header>
+
+                <p>
+                    Mailbox Listener is a component in James Mailbox System.
+                    Every time, if there is an action applied on a mailbox(adding, deleting), or an email(adding, deleting, updating flags...).
+                    Then an event represented for that action is generated and delivered to all Listeners that have registered before.
```

Comment:
Then an event represented for that action is generated and delivered to all Listeners that have registered before.
=> 
 Then an event representing that action is generated and delivered to all the Listeners that had been registered before.